news/4.35/platform_isv.html

```diff
@@ -43,6 +43,38 @@ <h2>Platform and Equinox API</h2>
   <!-- *********************** SWT *********************** -->
   <tr>
     <td id="SWT" class="section" colspan="2"><h2>SWT Changes</h2></td>
+    <tr id="imageConstructorGcDrawer"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/pull/1734 -->
+      <td class="title">Constructor for Dynamically Scaled, Custom-Drawn Image</td>
+      <td class="content">
+        <p>
+          The <code>Image</code> class provides a constructor for creating an empty image based on a concrete size,
+          which is then usually filled programmatically via a <code>GC</code>. Such an image will be created according to a
+          fixed zoom value. When the image is requested in a different zoom via <code>getImageData(zoom)</code>, the
+          image will be raster-scaled, leading to blurry results.
+        </p>
+        <p>
+          The API has been extended as follows to address this limitation:
+        <ul>
+          <li>
+            A new constructor has been added to the <code>Image</code> class:<br>
+            <code>Image(Device device, ImageGcDrawer imageGcDrawer, int width, int height)</code><br>
+          <li>
+            The constructors accepts implementations of <code>ImageGcDrawer</code> defining the method:<br>
+            <code>drawOn(GC gc, int imageWidth, int imageHeight)</code>
+          </li>
+        </ul>
+        </p>
+        <p>
+          Such an image will call <code>imageGcDrawer.drawOn(...)</code> whenever the image data for a different
+          zoom are requested to generate that data in a size as required for the specified zoom.
```

Comment:
Take this one with a pinch of salt because I'm not a native speaker, but I think here "data" is a singular noun (as opposed to a _collective_ noun) and therefore using "_is_" sounds better (to me):

```suggestion
          zoom is requested to generate that data in a size as required for the specified zoom.
```
Additionally, you refer to _that_ data afterwards which, for me, is again _singular_.

Since I'm not a native speaker and I have no strong opinion about this: feel free to respond with a "I disagree" and keep it just as it is :-)

Reply:
You are right. I even felt that this is wrong when writing and thought about it but erroneously concluded that "data" is used as plural. Fixed with recent commit. Thanks for the catch! :-)